Add eric k's advice

Eric-Kn opened the pull request:
I am really hoping I did all of these steps correctly.

redrambles commented:
Hey @Eric-Kn !

We got your advice page - but we seem to be missing your css file (`css/eric-k-advice.css`). Have a look if it is perhaps still uncommitted by running `git status`. If you need to - add and commit that file. 

In your advice file - you have an absolute link to your css file - but this will only work on your computer. We want to use relative links instead - which would make your link: `../css/eric-k-advice.css'. [More about relative vs absolute links here.](https://faqs.skillcrush.com/article/238-relative-vs-absolute-urls)

Lastly - we are missing is a link to your advice page from the main index.html file, so that folks can find your words of wisdom, once your code is merged in. Students will typically add their link to the top or the bottom of that long list. :)

Once you're done, add and commit your changes with `git add -A` and `git commit -m "Your message here"` and then push your branch up once more `git push origin advice`, which will automatically update your pull request. Then we should be good to merge in your code!

Let us know if you have any questions!

Cheers,
Ann

Eric-Kn commented:
Hello Ann,

I thought I had done it all correctly the first time, but I didn't see all
of my changes on Github so I wasn't sure what I had done wrong and was
waiting to hear back. I gave it another try, updated my files with relative
links instead of absolute links. Then I added, committed, and pushed all of
the changes to Github. I can see the link to my advice page at the bottom
of both the code and the website and it seems to load the way I had
intended, but I don't know if there is still some other coding that only
works on my computer. I still don't have a great grasp of what I am doing
with Github, but I am picking it up as I go.  Please let me know if
everything is correct or what still needs to be changed.

Eric T. Kneisler


On Wed, Apr 22, 2020 at 3:06 PM Ann Cascarano <notifications@github.com>
wrote:

> Hey @Eric-Kn <https://github.com/Eric-Kn> !
>
> We got your advice page - but we seem to be missing your css file (
> css/eric-k-advice.css). Have a look if it is perhaps still uncommitted by
> running git status. If you need to - add and commit that file.
>
> In your advice file - you have an absolute link to your css file - but
> this will only work on your computer. We want to use relative links instead
> - which would make your link: `../css/eric-k-advice.css'. More about
> relative vs absolute links here.
> <https://faqs.skillcrush.com/article/238-relative-vs-absolute-urls>
>
> Lastly - we are missing is a link to your advice page from the main
> index.html file, so that folks can find your words of wisdom, once your
> code is merged in. Students will typically add their link to the top or the
> bottom of that long list. :)
>
> Once you're done, add and commit your changes with git add -A and git
> commit -m "Your message here" and then push your branch up once more git
> push origin advice, which will automatically update your pull request.
> Then we should be good to merge in your code!
>
> Let us know if you have any questions!
>
> Cheers,
> Ann
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/105-practice/pull/1096#issuecomment-617972381>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APDQ75NSWOKMYNHKTMFIWCLRN453LANCNFSM4MLQ2UWA>
> .
>

acodeinprogress commented:
Hey @Eric-Kn,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon
